events/importer/enkora.py

```diff
@@ -854,13 +1062,24 @@ def generate_documentation_md() -> str:  # noqa: C901
         Generate MarkDown document out of Enkora importing rules.
         :return: documentation string
         """
+        import git  # gitpython
         from snakemd import Inline, MDList, new_doc, Paragraph
```

Comment:
Change: These are both missing from `requirements-dev.in`/`requirements-dev.txt`. Please add them.

Reply:
Added into requirement-dev.